ballerina-shell/modules/shell-core/src/test/resources/testcases/evaluator/basics.quoted.json

```diff
@@ -45,11 +45,6 @@
     "code": "string 'üňĩćőđę_ňāɱȇ = \"John doe\"; 'üňĩćőđę_ňāɱȇ",
     "expr": "\"John doe\""
   },
-  {
-    "description": "Use unicode with escape in name.",
-    "code": "string 'unicode_\\u{2324} = \"Jane doe\"; 'unicode_\\u{2324}",
-    "expr": "\"Jane doe\""
-  },
```

Comment:
In the shell, we get 
`error: undefined symbol 'unicode_', 'unicode_⌤ // Variable is unable to be referenced after declaration` for this code sample. 

But, this sample compiles as expected in the compiler and  `'unicode_⌤` can be referenced after declaration.
Hence created #30021 for the shell issue.

Reply:
Reason is https://github.com/ballerina-platform/ballerina-lang/issues/30021#issuecomment-821957812